upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -133,6 +140,17 @@ func (e *InstanceTemplate) Find(c *fi.Context) (*InstanceTemplate, error) {
 			if ni.Subnetwork != "" {
 				actual.Subnet = &Subnet{Name: fi.String(lastComponent(ni.Subnetwork))}
 			}
+
+			acs := ni.AccessConfigs
+			if len(acs) > 0 {
+				if len(acs) != 1 {
+					return nil, fmt.Errorf("unexpected number of access configs: %d", len(acs))
+				}
+				if acs[0].Type != accessConfigOneToOneNAT {
+					return nil, fmt.Errorf("unexpected access type: %s", acs[0].Type)
+				}
```

Comment:
Maybe mention InstanceTemplate name in error messages.

Reply:
done!